NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj

```diff
@@ -60,6 +59,13 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'net451' OR '$(TargetFramework)' == 'net46' ">
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
+    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="1.1.3" />
+  </ItemGroup>
+
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'net461' ">
+    <!-- Need to update this to 2.1.1, but testing with the lower version. -->
+    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="1.1.3" />
```

Comment:
if using same version for all targets, you can put this outside of the condition.

Reply:
I'm leaving the condition only because I want to get this upgraded. 
I'm creating a new issue and referencing this line exactly.